src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -1,174 +0,0 @@
-

```

Comment:
Thanks for catching that.
Remove the entry in `map.json` - that's perfectly safe.
I think you can change the other two references to `Microsoft.PowerShell.Utility.psd1`.